nativescript-slides.ts

```diff
@@ -172,6 +190,16 @@ export class SlideContainer extends AbsoluteLayout {
 		});
 	}
 
+	public resetAndroidTranslucentFlags(): void {
+		if (this._androidTranslucentStatusBar === true) {
+			let window = app.android.startActivity.getWindow();
+			window.clearFlags(LayoutParams.FLAG_TRANSLUCENT_STATUS);
```

Comment:
Should this let window but up two lines or is window already available I'm getting a compiler error here.